src/components/DiningRoomTable/DiningRoomTable.tsx

```diff
@@ -0,0 +1,69 @@
+import './DiningRoomTable.css';
+
+import { Button, Modal, Paper, Table, TableBody, TableCell, TableContainer, TableHead, TableRow } from '@mui/material';
+import { useState } from 'react';
+
+import DiningRoomModal from '../DiningRoomModal/DiningRoomModal';
+
+const tableStatuses = {
+  free: 'FREE',
+  occupied: 'OCCUPIED',
+  toClean: 'TO CLEAN',
+};
+
+function createData(tableNumber: number, tableStatus: string, tableInfo: string, tableAlert: string) {
+  return { tableNumber, tableStatus, tableInfo, tableAlert };
+}
+
+export default function DiningRoomTable() {
+  const [rows, setRows] = useState([createData(1, tableStatuses.free, '', '')]);
+  const [open, setOpen] = useState(false);
+  const handleOpen = () => setOpen(true);
+  const handleClose = () => setOpen(false);
+
+  function createNewTable() {
+    const { tableNumber } = rows[rows.length - 1];
+    rows[rows.length] = createData(tableNumber + 1, tableStatuses.free, '', '');
+    setRows(rows);
+  }
+
+  return (
+    <div className="DiningRoomTable__table">
+      <TableContainer component={Paper}>
+        <Table sx={{ minWidth: 0 }}>
+          <TableHead>
+            <TableRow className="DiningRoomTable__table-head">
+              <TableCell style={{ color: 'white', fontWeight: '600' }}>Table number</TableCell>
```

Comment:
If there is no reason for it then stay with one way of styling elements. Use `sx` or `css` file with proper classes and use them